src/scaffold.ts

```diff
@@ -66,13 +97,19 @@ export const scaffoldProjectRequest = async (item?: KafkaCluster | KafkaTopic) =
     logUsage(UserEvent.ProjectScaffoldingAction, {
```

Comment:
Probably best to handle this logic outside of the `logUsage()` block to make it easier to read and update - what do you think of something like this?
```suggestion
    let itemType: string | undefined;
    if (templateRequestOptions?.templateName) {
      // only URIs will specify the templateName
      itemType = "uri"
    } else if (templateRequestOptions?.topic) {
      // no templateName, but we have a topic name so this must've come from a topic tree item
      itemType = "topic"
    } else if (templateRequestOptions?.bootstrap_server) {
      // no templateName, but we have a bootstrap_server (but no topic name) so this must've come from a Kafka cluster tree item
      itemType = "cluster"
    }
    logUsage(UserEvent.ProjectScaffoldingAction, {
      status: "template picked",
      templateName: pickedTemplate.spec!.display_name,
      itemType,
    });
```